go/oasis-node/cmd/stake/account.go

```diff
@@ -87,6 +88,12 @@ var (
 		Short: "Generate an amend_commission_schedule transaction",
 		Run:   doAccountAmendCommissionSchedule,
 	}
+
+	accountLedgerListCmd = &cobra.Command{
```

Comment:
Why is this command in the staking subcommand? This seems like something that is signer-specific.

The ledger based signer will not only be used for staking, but also for other kinds of transactions and signatures (e.g., entity registration). I would put this under a new subcommand, e.g. `oasis-node signer ledger list_devices` or something.

Reply:
Agreed. I wasn't sure where to put this, so I just added this here.